main/test/ca/uwaterloo/flix/util/FlixSuite.scala

```diff
@@ -90,10 +91,12 @@ class FlixSuite(incremental: Boolean) extends AnyFunSuite {
             // Evaluate the function.
             val result = run()
             // Expect the true value, if boolean.
-            if (result.isInstanceOf[java.lang.Boolean]) {
-              if (result != true) {
-                fail("Expected true, but got false.")
-              }
+            result match {
+              case res: java.lang.Boolean =>
+                if (res != true) {
```

Comment:
Do you not need to unbox here?

Reply:
I think it's automatic... But I may be wrong? I can put in an else-branch to check :)

Reply:
It just to be safe. I think res.value() or something should make it explicit.